docs/CMakeLists.txt

```
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+project(docs)
+
+find_package(Doxygen REQUIRED)
+find_program(SPHINXBUILD NAMES sphinx-build)
+if(${SPHINXBUILD} STREQUAL SPHINXBUILD-NOTFOUND)
```

You can just do:
```suggestion
if(NOT SPHINXBUILD)
```

as per:
> True if given a variable that is defined to a value that is not a false constant.

and
> False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND.

---

thanks :-)